### PR TITLE
NameChange

### PR DESCRIPTION
**Pull request by wiredinhp**
Changing the whole codebase from Jira link commenter to Auto link commenter.

### CODE REVIEW

**Comment by sbimochan** [comment]
Please keep a habbit of writer proper PR summary

File: .github/workflows/pr_comment.yml [hunk 1]
```diff
@@ -3,15 +3,15 @@ on: pull_request
 jobs:
   example_comment_pr:
     runs-on: ubuntu-latest
-    name: Auto jira link commenter
+    name: Auto auto link commenter
```
Comment (sbimochan) [inline .github/workflows/pr_comment.yml:6]: Two auto?

File: .github/workflows/pr_comment.yml [hunk 2]
```diff
@@ -3,15 +3,15 @@ on: pull_request
...
     steps:
       - name: Checkout
         uses: actions/checkout@v4
 
       - name: Comment PR
-        uses: sbimochan/jira-link-commenter@v3.2
+        uses: sbimochan/auto-link-commenter@v3.2
```
Comment (sbimochan) [inline .github/workflows/pr_comment.yml:12]: this won't work

**Comment by sbimochan** [comment]
Too many issues. Please have a habit of self reviewing before creating PRs.

**Comment by wiredinhp** [comment]
@sbimochan Hi actually it is my first PR I will review first now and then create a PR in some time.

**Pull request closed without merging**